concent_api/core/tests/test_core_views.py

```diff
@@ -195,7 +212,7 @@ def test_send_should_return_http_400_if_get_invalid_type_of_message(self):
 
         response_400 = self.client.post(
             reverse('core:send'),
-            data = dump(self.message_want_to_compute, settings.CONCENT_PRIVATE_KEY, settings.CONCENT_PUBLIC_KEY),
+            data = dump(self.want_to_compute, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY),
```

Comment:
You should add an `assert` that makes sure that the type of `self.want_to_compute` is indeed invalid.